cloudsmith/data_source_package_list.go

```diff
@@ -96,30 +100,27 @@ func dataSourcePackageListRead(d *schema.ResourceData, m interface{}) error {
 	return nil
 }
 
-func flattenPackages(packages *[]cloudsmith.Package) []interface{} {
-	if packages != nil {
-		pkgs := make([]interface{}, len(*packages))
-		for i, packageItem := range *packages {
-			log.Printf("[DEBUG] package: %s", packageItem.Name)
-			pkg := make(map[string]interface{})
-			pkg["repository"] = packageItem.Repository
-			pkg["namespace"] = packageItem.Namespace
-			pkg["name"] = packageItem.Name
-			pkg["slug"] = packageItem.Slug
-			pkg["slug_perm"] = packageItem.SlugPerm
-			pkg["format"] = packageItem.Format
-			pkg["version"] = packageItem.Version
-			pkg["is_sync_awaiting"] = packageItem.IsSyncAwaiting
-			pkg["is_sync_completed"] = packageItem.IsSyncCompleted
-			pkg["is_sync_failed"] = packageItem.IsSyncFailed
-			pkg["is_sync_in_progress"] = packageItem.IsSyncInProgress
-			pkg["is_sync_in_flight"] = packageItem.IsSyncInFlight
-			pkgs[i] = pkg
-		}
-
-		return pkgs
+func flattenPackages(packages []cloudsmith.Package) []interface{} {
+	pkgs := make([]interface{}, len(packages))
+	for i, packageItem := range packages {
+		log.Printf("[DEBUG] package: %s", packageItem.GetName())
+		pkg := make(map[string]interface{})
+		pkg["repository"] = packageItem.GetRepository()
+		pkg["namespace"] = packageItem.GetNamespace()
+		pkg["name"] = packageItem.GetName()
+		pkg["slug"] = packageItem.GetSlug()
+		pkg["slug_perm"] = packageItem.GetSlugPerm()
+		pkg["format"] = packageItem.GetFormat()
+		pkg["version"] = packageItem.GetVersion()
+		pkg["is_sync_awaiting"] = packageItem.GetIsSyncAwaiting()
+		pkg["is_sync_completed"] = packageItem.GetIsSyncCompleted()
+		pkg["is_sync_failed"] = packageItem.GetIsSyncFailed()
+		pkg["is_sync_in_progress"] = packageItem.GetIsSyncInProgress()
+		pkg["is_sync_in_flight"] = packageItem.GetIsSyncInFlight()
```

Comment:
NAB: Should we put a status in for `is_quarantined` (ofc that's an enhancement 😆)

Reply:
Yep, will do in a follow up!